ipv-stub/src/endpoints/ipv-authorize.ts

```diff
@@ -34,13 +36,45 @@ async function get(
     throw new CodedError(400, "Query string parameters are null");
   }
 
-  //This is a first pass, will be changed to encrypted
-  const plaintextJwt = event.queryStringParameters["request"] as string;
-  if (!plaintextJwt) {
+  const publicKeyPem = `-----BEGIN PUBLIC KEY-----
+      MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApflLYqZm5IawLAHYtWoU
+  vKdO7cFBmDIOASlGgGCEG0PVBC4FJH2pM3FUw72n7YTS+H73Y8ZfTNIgu9K7zxEa
+  mCwimUAKU8Lsjq6Pqa0pZr2rE4l2MfO2j91uCcdlTzdM0kOkwcbzwqEdbDU+FJ4x
+  FT5aaOWyuN/BKFsc5kNz2t4+OaeRu/ev3h7WCqh2MMW5PWDbR2lBnKZR8HvuXZc5
+  ay0dUx098UjkLEBHIyT3FfzhXFMF2ZdOSysDMa64KwqeAWs6tjwM9+Bp3DYLkTsx
+  BML/eqgIwdZI5QBCTxD8YC2oxC1obMspiAoEz05wt8cYscmT4rZAdBGMspuxNqo4
+  6wIDAQAB
+  -----END PUBLIC KEY-----`;
+
+// Step 2: Convert the Base64-encoded data to a Uint8Array for encryption
+  const dataToEncrypt = Uint8Array.from(`eyJhbGciOiJzb21lLWFsZyJ9.eyJmb28iOiJiYXIifQ==.eyJ0aGlzIjoiZG9lc24ndG1hdHRlciJ9`, c => c.charCodeAt(0));
+
+// Step 3: Import the public key
+  const publicKey = await importSPKI(publicKeyPem, "RSA-OAEP-256");
+
+// Encrypt the data
+  const jwe = await new CompactEncrypt(dataToEncrypt)
```

Comment:
What's this for?

Reply:
sorry left in by accident just taken out